tests/code/test_search.py

```diff
@@ -29,33 +29,33 @@ def test_searchStudents():
     #tests that the search works
 
     query = 'z'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert searchResults['Zach Neill (neillz)'] == 'Zach Neill (neillz)'
 
 
     #test for last name
     query = 'khatt'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert searchResults['Sreynit Khatt (khatts)'] == 'Sreynit Khatt (khatts)'
 
     #test with non existing username
     query = 'abc'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     query = 'its easy as'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     query = '123'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     #test with multiple users matching query
     with mainDB.atomic() as transaction:
         User.get_or_create(username = 'sawconc', firstName = 'Candace', lastName = 'Sawcon', bnumber = '021556782', isStudent = True)
         query = 'sa'
-        searchResults = searchUsers(query)
+        searchResults = searchUsers(query, False)
         assert len(searchResults) == 2
```

Comment:
I don't see any tests with the new parameter being True. You could also add a default value, which would let you not update all of the existing function calls.